pkg/model/dfw/rule.go

```diff
@@ -321,6 +322,7 @@ func (f *FwRule) originalRuleComponentsStr() []string {
 		f.getDstString(),
 		// todo: origRuleObj.Services is not always the services, can also be service_entries
 		f.getShortPathsString(f.OrigRuleObj.Services),
+		f.Conn.String(),
```

Comment:
this format is used to print the original nsx config, but `f.Conn.String()` is not a valid nsx input config..  it contains the details of how the services input is interpreted by the analyzer..
maybe this should be in a separate log message.

Reply:
removed, the connection is part of the effective rule later